financial-connections/src/main/java/com/stripe/android/financialconnections/features/error/ErrorPreviewParameterProvider.kt

```diff
@@ -0,0 +1,76 @@
+@file:Suppress("LongMethod")
+
+package com.stripe.android.financialconnections.features.error
+
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import com.airbnb.mvrx.Loading
+import com.airbnb.mvrx.Success
+import com.stripe.android.core.exception.APIException
+import com.stripe.android.financialconnections.exception.InstitutionPlannedDowntimeError
+import com.stripe.android.financialconnections.exception.InstitutionUnplannedDowntimeError
+import com.stripe.android.financialconnections.model.FinancialConnectionsInstitution
+
+internal class ErrorPreviewParameterProvider :
```

Comment:
More related to the error screen -> added some previews and snapshot tests that were missing.